mon-pix/app/components/tutorial-item.js

```diff
@@ -43,11 +44,15 @@ export default class TutorialItemComponent extends Component {
   }
 
   get buttonText() {
-    return this.savingStatus === buttonStatusTypes.recorded ? 'Retirer' : 'Enregistrer';
+    return this.savingStatus === buttonStatusTypes.recorded ?
+      this.intl.t('pages.user-tutorials.tutorial.actions.remove.title') :
+      this.intl.t('pages.user-tutorials.tutorial.actions.save.title');
   }
 
   get buttonTitle() {
-    return this.savingStatus === buttonStatusTypes.recorded ? 'Retirer' : 'Enregistrer dans ma liste de tutos';
+    return this.savingStatus === buttonStatusTypes.recorded ?
+      this.intl.t('pages.user-tutorials.tutorial.actions.remove.title') :
+      this.intl.t('pages.user-tutorials.tutorial.actions.save.description');
```

Comment:
Ça me semble étrange d'être dans `buttonTitle` et d'utiliser une clef `description`. Peut-être qu'il y a un renommage un peu plus global à faire avec `buttonText`

Reply:
J'ai fait un renommage, est-ce que ça te paraît mieux?

Reply:
Dans cette logique, je pense qu'il faudrait faire 

```suggestion
  get buttonDescription() {
    return this.savingStatus === buttonStatusTypes.recorded ?
      this.intl.t('pages.user-tutorials.tutorial.actions.remove.description') :
      this.intl.t('pages.user-tutorials.tutorial.actions.save.description');
```

et du coup 
```suggestion
 get buttonContent() {
    return this.savingStatus === buttonStatusTypes.recorded ?
      this.intl.t('pages.user-tutorials.tutorial.actions.remove.content') :
      this.intl.t('pages.user-tutorials.tutorial.actions.save.content');
 }
```

Ça ouvre la question suivante au final : "est-ce l'utilisation d'un mot à des endroits différents marque la même intention ?"
Par extension, est-ce qu'on est sur que l'utilisation de ce même mot restera toujours la même ? (YAGNI — You ain't gonna need it — mis à part -> il s'agit vraiment de dissocier le mot de l'intention)